tools/distribution/src/utils.py

```diff
@@ -71,9 +71,11 @@ def read_sdk_version() -> str:
     Reads SDK version from 'Sources/Datadog/Versioning.swift'.
     """
     file = 'DatadogCore/Sources/Versioning.swift'
-    version_regex = r'^.+\"([0-9]+\.[0-9]+\.[0-9]+[\-a-z0-9]*)\"'  # e.g. 'internal let __sdkVersion = "1.8.0-beta1"'
 
-    versions: [str] = []
+    # https://packaging.pypa.io/en/latest/version.html#packaging.version.VERSION_PATTERN
+    version_regex = re.compile(r'^.+\"' + VERSION_PATTERN + r'\"$', re.VERBOSE | re.IGNORECASE) # e.g. 'internal let __sdkVersion = "1.8.0-beta.1"
```

Comment:
for such trivial case, we should use simply parse the string than taking a dependency 

1.8.0-beta1
```
split by -
- 1.8.0
  - split by .
    - major: 1
    - minor: 8
    - patch: 0 
- beta1
    - read from back until there is digit
      - beta
      - 1
```

Reply:
The dependency was already introduced in 856ed98b034713fc340452ff8d26a2ebc3834dec and I think it is best to rely on the same pattern everywhere 👍